README.md

```diff
@@ -94,8 +94,9 @@ the following requirements are applied as well:
 
 - A landscape image is preferred.
 - Aspect ratio should respect the logo of the brand.
-- Shortest side of the image must be at least 128 pixels, 256 pixels for the hDPI version.
-- Shortest side of the image must be not bigger than 256 pixels, 512 pixels for the hDPI version.
+- Image length
```

Comment:
It is not the length.... it is the width.

Reply:
Width implies an assumption that the image is longest on its horizontal axis, but it could also be in the height.